playground/frontend/playground_components/lib/widgets/split_view/controller.dart

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'package:flutter/material.dart';
+import '../../constants/sizes.dart';
```

Comment:
Auto-formatting should add a blank line above. Be sure to auto-format.

Reply:
formatting the file doesn't add a blank line. My default formatter for Dart is set to: `"editor.defaultFormatter":"Dart-Code.dart-code"`

Reply:
Oh, so it's Android Studio does this in addition to the Dart formatter. It works with just `Ctrl-Alt-L` there.
And there is an issue for this: https://github.com/dart-lang/dart_style/issues/1120